src/compiler/checker.ts

```diff
@@ -4738,6 +4739,14 @@ namespace ts {
             return checkTypeAssignableTo(source, target, /*errorNode*/ undefined);
         }
 
+        /**
+         * This is *not* a bi-directional relationship.

```

Comment:
Is this true of `isTypeAssignableTo` too? Its former usage in this diff indicates 'yes'.